bin/git-forgit

```diff
@@ -264,7 +305,7 @@ _forgit_diff() {
     done
     opts="
         $FORGIT_FZF_DEFAULT_OPTS
-        +m -0 --bind=\"enter:execute($FORGIT diff_enter {} $escaped_commits | $_forgit_enter_pager)\"
+        +m -0 --bind=\"enter:execute($FORGIT diff_enter {} $escaped_commits | $FORGIT enter_page)\"
```

Comment:
Typo, I think `_forgit_enter_pag` matches the other changes you made here

Reply:
I think this is actually correct this way. I've added `enter_page` to our `private_commands` array, so executing `$FORGIT enter_page` will execute the main script with the known `enter_page` argument, which in turn leads to `_forgit_enter_page` being executed. We can not call `_forgit_enter_page` directly because fzf executes the code snippet in question on a subshell. We do the same with our preview functions.